### PR TITLE
Liquify doc fixes

### PR DESCRIPTION
**Pull request by moy**

### CODE REVIEW

File: src/iop/liquify.c [hunk 1]
```diff
@@ -3324,17 +3324,19 @@ void gui_init (dt_iop_module_t *module)
 
   gtk_box_pack_start(GTK_BOX(module->widget), hbox, TRUE, TRUE, 0);
 
-  dt_liquify_layers[DT_LIQUIFY_LAYER_PATH].hint           = _("ctrl-click to add node\nright click to remove path");
-  dt_liquify_layers[DT_LIQUIFY_LAYER_CENTERPOINT].hint    = _("click and drag to move - click : linear or feathered\n"
-                                                              "ctrl-click : autosmooth, cusp, smooth, symmetrical"
+  dt_liquify_layers[DT_LIQUIFY_LAYER_PATH].hint           = _("ctrl-click to add node\n"
+                                                              "right click to remove path\n"
+                                                              "ctrl-alt-click to toggle line/curve");
```
Comment (TurboGit) [inline src/iop/liquify.c]: No we don't want 3 line hints messages. In some DE there is not enough room at the top and this makes the whole darkroom to be resized and so move when hovering the corresponding item. Maybe keeping the two first on the same line?

File: src/iop/liquify.c [hunk 2]
```diff
@@ -3324,17 +3324,19 @@ void gui_init (dt_iop_module_t *module)
...
+                                                              "ctrl-click: autosmooth, cusp, smooth, symmetrical"
                                                               " - right click to remove");
   dt_liquify_layers[DT_LIQUIFY_LAYER_CTRLPOINT1].hint     = _("drag to change shape of path");
   dt_liquify_layers[DT_LIQUIFY_LAYER_CTRLPOINT2].hint     = _("drag to change shape of path");
   dt_liquify_layers[DT_LIQUIFY_LAYER_RADIUSPOINT].hint    = _("drag to adjust warp radius");
   dt_liquify_layers[DT_LIQUIFY_LAYER_HARDNESSPOINT1].hint = _("drag to adjust hardness (center)");
   dt_liquify_layers[DT_LIQUIFY_LAYER_HARDNESSPOINT2].hint = _("drag to adjust hardness (feather)");
   dt_liquify_layers[DT_LIQUIFY_LAYER_STRENGTHPOINT].hint  = _("drag to adjust warp strength\n"
-                                                              "ctrl-click : linear, grow, and shrink");
+                                                              "ctrl-click: linear, grow, and shrink");
```
Comment (TurboGit) [inline src/iop/liquify.c]: ok

File: doc/usermanual/darkroom/modules/correction/liquify.xml [hunk 3]
```diff
@@ -288,7 +288,7 @@
               </row>
               <row>
                 <entry>
-                  <emphasis>cups</emphasis>
+                  <emphasis>cusp</emphasis>
```
Comment (TurboGit) [inline doc/usermanual/darkroom/modules/correction/liquify.xml]: good catch!

File: src/iop/liquify.c (hunk 1 above)
Comment (moy) [inline src/iop/liquify.c]: OK. While I'm there, maybe replace `to` with `:` to gain a few characters and for consistency with most other strings. That'd give:

```
"ctrl-click: add node, right click: remove path\n"
"ctrl-alt-click: toggle line/curve"
```

Comment (TurboGit) [inline src/iop/liquify.c]: Sounds good to me!

**Comment by moy** [comment]
OK, I applied the requested changes and rebased onto master. Should be all right now.

**Comment by TurboGit** [comment]
Merged thanks.

**Pull request merged**